dsl/src/main/java/tech/pegasys/peeps/node/GoQuorum.java

```diff
@@ -0,0 +1,169 @@
+    final List<String> entryPoint = Lists.newArrayList("/bin/sh", "-c");
+    final String initCmd =
+        "mkdir -p '"
+            + DATA_DIR
+            + "/geth' && geth --datadir \""
+            + DATA_DIR
+            + "\" init "
+            + CONTAINER_GENESIS_FILE
+            + " && echo '##### GoQuorum INITIAILSED #####' && ";
+
+    addNodePrivateKey(config, commandLineOptions, container);
+    //    if (config.isPrivacyEnabled()) {
+    //      addPrivacy(config, commandLineOptions, dockerContainer);
+    //    }
+
+    final String goCommandLine = initCmd + "geth " + String.join(" ", commandLineOptions);
+    LOG.info("GoQuorum command line: {}", goCommandLine);
+
+    entryPoint.add(goCommandLine);
+
+    container
+        .withCreateContainerCmdModifier(cmd -> cmd.withEntrypoint(entryPoint))
+        .waitingFor(liveliness());
+  }
+
+  @Override
+  public String getLogs() {
+    return DockerLogs.format("GoQuorum", container);
+  }
+
+  private AbstractWaitStrategy liveliness() {
+    return Wait.forLogMessage(".*endpoint=0.0.0.0:8545.*", 1);
+  }
+
+  private List<String> standardCommandLineOptions() {
+    return Lists.newArrayList(
+        "--verbosity",
+        "5",
+        "--rpccorsdomain",
+        "\"*\"",
+        // "--mine",
+        // "--miner.etherbase",
+        // "1b23ba34ca45bb56aa67bc78be89ac00ca00da00",
+        "--rpc",
+        "--rpcaddr",
+        "\"0.0.0.0\"",
+        "--rpcport",
+        "8545",
+        "--rpcapi",
+        "admin,debug,web3,eth,txpool,personal,clique,miner,net",
+        "--ws",
+        "--debug");
+  }
+
+  private void addBootnodeAddress(
+      final Web3ProviderConfiguration config, final List<String> commandLineOptions) {
+    config
+        .getBootnodeEnodeAddress()
+        .ifPresent(
+            enode -> {
+              if (!enode.isEmpty()) {
+                commandLineOptions.addAll(Lists.newArrayList("--bootnodes", enode));
+              }
+            });
+  }
+
+  private void addContainerNetwork(
+      final Web3ProviderConfiguration config, final GenericContainer<?> container) {
+    container.withNetwork(config.getContainerNetwork());
+  }
+
+  private void addCorsOrigins(
+      final Web3ProviderConfiguration config, final List<String> commandLineOptions) {
+    config
+        .getCors()
+        .ifPresent(cors -> commandLineOptions.addAll(Lists.newArrayList("--rpccorsdomain", cors)));
+  }
+
+  private void addNodePrivateKey(
+      final Web3ProviderConfiguration config,
+      final List<String> commandLineOptions,
+      final GenericContainer<?> container) {
+    container.withClasspathResourceMapping(
+        config.getNodeKeyPrivateKeyResource().get(),
+        CONTAINER_NODE_PRIVATE_KEY_FILE,
+        BindMode.READ_ONLY);
+    commandLineOptions.addAll(Lists.newArrayList("--nodekey", CONTAINER_NODE_PRIVATE_KEY_FILE));
+  }
+
+  //  private void addPrivacy(
```

Comment:
Same as above, assuming this can be removed until Privacy config is added in?

Reply:
_sort of_ like it here as a statement that it needs to be done, but .... can be convinced.